app/assets/stylesheets/scm.css.sass

```diff
@@ -77,6 +77,10 @@ li.change
 .repository--revision-toolbar
   margin-top: 3rem
 
+.toolbar-items
+  #repository-checkout-url
+    min-width: 350px
```

Comment:
Again I'd aprechiate a comment indicating that this rule is included for a single browser only...

Especially for CSS hacks there is a high chance to eliminate them as long as we know for which browsers they were performed.